core/rawdb/database.go

```diff
@@ -173,6 +173,8 @@ func InspectDatabase(db ethdb.Database, keyPrefix, keyStart []byte) error {
 			preimages.Add(size)
 		case bytes.HasPrefix(key, configPrefix) && len(key) == (len(configPrefix)+common.HashLength):
 			metadata.Add(size)
+		case bytes.HasPrefix(key, upgradeConfigPrefix) && len(key) == (len(configPrefix)+common.HashLength):
+			metadata.Add(size)
```

Comment:
Commenting that I agree we probably call this `metadata`.

Reply:
This just follows the existing pattern.